lib/flat_search_web/templates/flat/page_live.html.leex

```diff
@@ -1,2 +1,3 @@
 <%= render("_search.html", assigns) %>
+<p class="mb-4 text-gray-500"><%= length(assigns.flats) %> flats found</p>
```

Comment:
generally, you should refer to param from assigns by assings[:flats]
but this method is a kinda failsafe when a param can(but not always) exist, so I think just length(@flats) would be ok here since on mount you already initializing this @flats param

Reply:
RIght, since assings[:flats] covers 2 cases, when a key is an atom and when it isn't.

Reply:
The main reason for using assings[:flats] is that won't error, it just returns nil if the key is not found inside the map. In opposition to assings.flats, which will error when the key is not found.